third_party/terraform/tests/resource_compute_forwarding_rule_test.go.erb

```diff
@@ -182,3 +182,59 @@ resource "google_compute_forwarding_rule" "foobar" {
 }
 `, poolName, ruleName)
 }
+
+func TestAccComputeForwardingRule_mirror(t *testing.T) {
+	t.Parallel()
+
+	backendName := fmt.Sprintf("tf-%s", acctest.RandString(10))
+	healthcheckName := fmt.Sprintf("tf-%s", acctest.RandString(10))
+	ruleName := fmt.Sprintf("tf-%s", acctest.RandString(10))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeForwardingRuleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccComputeForwardingRule_mirror(backendName, healthchechName, ruleName),
+			},
+
+			{
+				ResourceName:      "google_compute_forwarding_rule.foobar",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+<% unless version == 'ga' -%>
+func testAccComputeForwardingRule_mirror(backendName, healthchechName, ruleName string) string {
+	return fmt.Sprintf(`
+resource "google_compute_region_backend_service" "foobar-be" {
+  name        	    = "%s"
```

Comment:
Can you make the `=` signs in this config line up by removing/adding spaces? Just a small detail that makes it a little easier to read

Reply:
Thanks, fixed